interface/interface.dm

```diff
@@ -116,6 +116,7 @@ Hotkey-Mode: (hotkey-mode must be on)
 \tt = say
 \to = OOC
 \tb = resist
+\t<B></B>h = stop pulling

```

Comment:
what's with the empty tag?

Reply:
It has to be done so byond doesn't try to treat the \th as an escape character. It's the same method that was used for the talk-wheel hotkey.